### PR TITLE
Add additional python requirements

### PR DESCRIPTION
**Pull request by garymm**
These are used by some of the python code in the package, e.g., https://github.com/microsoft/onnxruntime/blob/0292356bd7ce5b379be8ee9820bfbf4fd321ebc5/onnxruntime/python/tools/transformers/optimizer.py#L25

https://github.com/microsoft/onnxruntime/blob/c8270c2940f72a3722891fec7edf3c9ac542f904/onnxruntime/python/tools/symbolic_shape_infer.py#L10

https://github.com/microsoft/onnxruntime/blob/0292356bd7ce5b379be8ee9820bfbf4fd321ebc5/onnxruntime/python/tools/transformers/torch_onnx_export_helper.py#L9

### CODE REVIEW

File: requirements.txt.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
+coloredlogs
```
Comment (garymm) [inline requirements.txt.in:1]:
```suggestion
coloredlogs
sympy
```

File: requirements.txt.in [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
```
Comment (garymm) [inline requirements.txt.in:1]:
```suggestion
coloredlogs
packaging
```

**Review by thiagocrepaldi — DISMISSED** [review]
I believe you need to add its license at ThirdPartyNotice.txt

Refer to the commit below as an example

commit ef20efe015dfd725445e801f389c429706a1c99b
Author: Thiago Crepaldi <thiago.crepaldi@microsoft.com>
Date:   Mon Aug 17 15:03:54 2020 -0700

    Register cerberus license into ThirdPartyNotices.txt (#4828)

You also have to ensure the  Governance Compliance component check is passing (the CI does not fail when an issue is found, but it is logged instead)

**Comment by baijumeswani** [comment]
@snnn does the file need to be named as requirements.txt (without the `.in` suffix) for component governance stuff?

**Comment by thiagocrepaldi** [comment]
> Thanks @thiagocrepaldi .
> 
> > I believe you need to add its license at ThirdPartyNotice.txt
> 
> We're not distributing this software ourselves (e.g. by including any part of it in our packages), just declaring deps, so I don't think so. Notice for example that numpy is not mentioned in that file. But if I'm wrong could you please explain what the criteria is for being included in that file?
> 
AFAIK `cerberus`  and other projects are not redistributed as well, but I don't remember the exact criterium. Someone reached out to me at the time and asked me to do for compliance. Are `tensorboard`, `tensorboardX` or `mpi4py` distributed by us? They are in the aforementioned file.

> > You also have to ensure the Governance Compliance component check is passing (the CI does not fail when an issue is found, but it is logged instead)
> 
> I don't see that in the list of checks. Do I need to trigger it manually? How?
IIRC it is part of `Linux CPU Pipeline`. Possibly the `Component Detection` step right [here](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=650125&view=logs&j=7536d2cd-87d4-54fe-4891-bfbbf2741d83&t=a1755116-24b0-5f20-37f0-552538b7a95a)

**Comment by thiagocrepaldi** [comment]
> > Thanks @thiagocrepaldi .
> > > I believe you need to add its license at ThirdPartyNotice.txt
> > 
> > 
> > We're not distributing this software ourselves (e.g. by including any part of it in our packages), just declaring deps, so I don't think so. Notice for example that numpy is not mentioned in that file. But if I'm wrong could you please explain what the criteria is for being included in that file?
> 
> AFAIK `cerberus` and other projects are not redistributed as well, but I don't remember the exact criterium. Someone reached out to me at the time and asked me to do for compliance. Are `tensorboard`, `tensorboardX` or `mpi4py` distributed by us? They are in the aforementioned file.
> 
> > > You also have to ensure the Governance Compliance component check is passing (the CI does not fail when an issue is found, but it is logged instead)
> > 
> > 
> > I don't see that in the list of checks. Do I need to trigger it manually? How?
> > IIRC it is part of `Linux CPU Pipeline`. Possibly the `Component Detection` step right [here](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=650125&view=logs&j=7536d2cd-87d4-54fe-4891-bfbbf2741d83&t=a1755116-24b0-5f20-37f0-552538b7a95a)

Just talked to @snnn and he said `(only) The ones we ship in our packages or docker images`

**Comment by snnn** [comment]
> @snnn does the file need to be named as requirements.txt (without the `.in` suffix) for component governance stuff?

We use this file to generate a  requirements.txt in the build folder. I believe component governance scans build folder as well. So, it's fine.

**Comment by garymm** [comment]
Can someone please approve or critique?

**Review by thiagocrepaldi — APPROVED** [review]
LGTM

**Pull request merged**